app/Server/src/SERVERAPI/ViewComponents/AnimalsViewComponent.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Agri.Data;
+using Agri.Models;
+using Agri.Models.Configuration;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using SERVERAPI.Models.Impl;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SERVERAPI.ViewComponents
+{
+    public class Animals : ViewComponent
+    {
+        private IHostingEnvironment _env;
```

Comment:
Remove the IHostingEnvironment here and in the constructor.  I went through the project and removed them from all the Controllers.  They were never used in the code.